### DISCUSSION TITLE
Parse issue numbers from filename

### DISCUSSION THREAD

**Discussion by francispeixoto** [gotson/komga#1728] [Ideas]
Komga will assume that any newly imported file in a series is the next issue, even if there is a gap in the actual issue number.

ex: My collection will have issues 1 to 7, and I receive issue 10, but komga will tag it as issue 8 because in its database, the next missing issue is 8. The filename has the correct issue number, but Komga ignores it. 

This means "on deck" will incorrectly tell me the next issue is available, because in reality, there is a gap in the actual issues.

**Comment by gotson** [comment]
Komga relies on the ordering of files in the directory. There's no plans to add any filename parsing capability.
